packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_fr_FR.json

```
@@ -33,10 +33,31 @@
   "pci_projects_project_instances_add_script_add_label": "Ajouter",
-  "pci_projects_project_instances_add_privateNetwork_label": "Réseau privé attaché",
+
+  "pci_projects_project_instances_add_privateNetwork_title": "Configurez votre réseau",
+  "pci_projects_project_instances_add_privateNetwork_sub_title": "Merci de sélectionner le type de connexion réseau de votre nouvelle Instance du Cloud Public.",
+  "pci_projects_project_instances_add_privateNetwork_info1": "Le mode Public est sélectionné par défaut : il s'agit du modèle de réseau classique. Il est possible que ce ne soit pas compatible avec tous les produits ni disponible dans toutes les régions.",
+  "pci_projects_project_instances_add_privateNetwork_public_mode": "Mode Public",
+  "pci_projects_project_instances_add_privateNetwork_public_mode_description1": "C'est le modèle de réseau classique des instances Public Cloud. Les instances auront un port réseau public rattaché.",
+  "pci_projects_project_instances_add_privateNetwork_public_mode_description2": "Veuillez noter que ce mode n'est pas compatible pour les nouveaux services de réseau (Load Balancer or les IP flottantes).",
+  "pci_projects_project_instances_add_privateNetwork_private_mode": "Mode Privé",
+  "pci_projects_project_instances_add_privateNetwork_private_mode_description1": "Les instances seront uniquement attachées à un réseau privé. Il peut être soit complètement privé, soit exposé à Internet en utilisant les services Gateway ou Load Balancer avec les IP flottantes.",
+  "pci_projects_project_instances_add_privateNetwork_private_mode_description2": "Veuillez noter que pour un accès ssh depuis internet, une IP flottante ou une autre instance (appelée ssh-proxy) est nécessaire.",
+  "pci_projects_project_instances_add_privateNetwork_label": "Sélectionner un réseau privé",
+  "pci_projects_project_instances_add_privateNetwork_placeholder": "Sélectionnez un réseau privé",
   "pci_projects_project_instances_add_privateNetwork_none": "Aucun",
-  "pci_projects_project_instances_add_privateNetwork_description": "Par défaut les instances Public Cloud possèdent un réseau public, vous pouvez ajouter un réseau privé.",
-  "pci_projects_project_instances_add_privateNetwork_add": "Créer un réseau privé",
+  "pci_projects_project_instances_add_privateNetwork_description": "Dans ce mode l'instance est attachée à l'un des réseaux privé locaux.",
+  "pci_projects_project_instances_add_privateNetwork_add": "Créer un nouveau réseau privé",
+  "pci_projects_project_instances_add_privateNetwork_gateway_detectected_banner_info1": "Nous avons détecté une Gateway à l'intérieur du réseau privé sélectionné. Cette dernière n’étant pas compatible avec le mode de réseau concerné, cela peut entraîner un comportement réseau inattendu.",
+  "pci_projects_project_instances_add_privateNetwork_gateway_detectected_banner_info2": "Si vous souhaitez connecter une instance à ce réseau privé, veuillez sélectionner <strong>le mode privé</strong>.",
```

To avoid HTML into content

```suggestion
  "pci_projects_project_instances_add_privateNetwork_gateway_detectected_banner_info2": "Si vous souhaitez connecter une instance à ce réseau privé, veuillez sélectionner {{strongTag}}le mode privé{{strongTagEnd}}.",
```

then as sample:
`<p data-translate="my_key_with_tag" data-translate-values="{ strongTag: '<strong>', strongTagEnd: '</strong>' }">`

---

As we discuss not unfortunately not feasible :(.
So can be skipped.